lib/Dialect/Ibis/Transforms/IbisTunneling.cpp

```diff
@@ -296,7 +296,8 @@ LogicalResult Tunneler::tunnelDown(InstanceGraphNode *currentContainer,
   llvm::DenseMap<StringAttr, OutputPortOp> outputPortOps;
   for (PortInfo &pi : portInfos) {
     outputPortOps[pi.portName] = rewriter.create<OutputPortOp>(
-        op.getLoc(), circt::hw::InnerSymAttr::get(pi.portName), pi.getType());
+        op.getLoc(), pi.portName, circt::hw::InnerSymAttr::get(pi.portName),
```

Comment:
Unrelated to this PR, but will `circt::hw::InnerSymAttr::get(pi.portName)` create a unique symbol?